bedboss/bedqc/bedqc.py

```diff
@@ -0,0 +1,105 @@
+import logging
+import os
+import tempfile
+import pypiper
+import subprocess
+
+from bedboss.const import MAX_FILE_SIZE, MAX_REGION_SIZE, MIN_REGION_WIDTH
+from bedboss.exceptions import QualityException
+
+
+_LOGGER = logging.getLogger("bedboss")
+
+
+def bedqc(
+    bedfile: str,
+    outfolder: str,
+    max_file_size: int = MAX_FILE_SIZE,
+    max_region_size: int = MAX_REGION_SIZE,
```

Comment:
I think this is the wrong name for this variable. It's not about region size, it's about number of regions, right?